.babelrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This is necessary due to https://github.com/danger/danger-js/issues/261, and can be removed once Jest releases 20.1.0 _or_ danger-js updates its dependency on jest to 20.1.0-alpha.3.

I don't want to merge this PR until we can avoid using this workaround.